### ISSUE TITLE
Move plugin to Community Repository

### ISSUE THREAD

**Issue by RubenKelevra** [sabeechen/hassio-google-drive-backup#425]
Love this plugin! I feel like this should be included in the [Community Repository](https://addons.community/) for better visibility!

Can you move it?

**Comment by sabeechen** [comment]
tl;dr For the immediate future I think the addon is just going to stay as some project that people stumble into, as it presently can't afford substantial growth.  I'll reassess that once its in a more financially stable place.

I might try to put it the community repo someday.  The problem right now is that because of how Google Drive handles authentication the server for the project costs about $2 a day to run and that number will increase if more people start using the addon.  Right now I'm getting donations that more than cover the ongoing costs, but eventually the donations will dry up.  Based on the growth of the project and the rate of donations (which I expect to dwindle off over time) its reasonable to think the project will run out of money in 2-ish years.  At that point I'd either have to fund it out of pocket or get more aggressive about asking for money, both of which Id like to avoid.

I'd also expect that if the project "looked like" part of the Home Assistant project rather than being an individual contribution, people would be less inclined to donate.  Tragedy of the commons and all that.

Long term financial sustainability of the project is something I think about, and I have some plans to address parts of it:
- I'm looking to switch hosting providers for the web-based component of the addon.  Right now I'm hosting on Google Cloud Run, which was very quick 'n easy to set up and very reliable but I've now learned is horrendously overpriced once you hit any serious utilization.  This could extend the runway considerably.
- I plan to encourage using custom credentials to connect to Google Drive which mitigates the need to use the web-based part of the addon for authentication, thus using my expensive server less.

**Comment by lordratner** [comment]
I agree this should be integrated fully into HA. It's a tremendously nice add-on. Thanks.

For the service costs, do you pay per user setup or monthly per user? Basically, would it save you money if I reauthenticated the hard way? I'm familiar with the process from other integrations. 

You might consider a "recommended donation" amount, or at least explaining what it costs you to maintain the service in the donation section. I would have given you more than $5, and will do so now that I know what you're paying. 

Either way, hopefully by the time you run out of funds, this will be absorbed into HA.

**Comment by dougle03** [comment]
> Literally pennies per person

Hopefully my paypal donation goes a small way to helping you to keep the service going. Saved me too many times to mention. Thanks very much.
